src/view/invoice.js

```diff
@@ -38,13 +46,22 @@ const InvoiceView = ({ store, nav, invoice }) => (
     <MainContent>
       <Card>
         <BalanceLabel style={styles.balance}>
+          <BalanceLabelNumeral
+            style={[
+              styles.fiatUnit,
+              { fontSize: store.settings.displayFiat ? font.sizeXXXL : 0 },
+            ]}
+          >
+            $
+          </BalanceLabelNumeral>
           <AmountInputField
             autoFocus={true}
-            value={store.invoice.amount}
-            onChangeText={amount => invoice.setAmount({ amount })}
+            onChangeText={amount => setAmount(amount, invoice, store.settings)}
```

Comment:
Please revert back `store.invoice.amount` should reflect what actually rendered on screen. The actual satoshi value that is passed to the grpc api should be handled in a different variable.